launcher/Application.cpp

```diff
@@ -746,11 +744,7 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
     }
 
     // Themes
-    {
-        m_themeManager = new ThemeManager(m_mainWindow);
-
-        m_themeManager->InitializeThemes();
-    }
+    m_themeManager = std::make_unique<ThemeManager>(ThemeManager(m_mainWindow));
```

Comment:
here you just need to do `m_themeManager = std::make_unique<ThemeManager>(m_mainWindow);`